src/ReachSets/discretize.jl

```diff
@@ -78,23 +79,23 @@ This uses a first order approximation of the ODE, and matrix norm upper bounds,
 see Le Guernic, C., & Girard, A., 2010, *Reachability analysis of linear systems
 using support functions. Nonlinear Analysis: Hybrid Systems, 4(2), 250-262.*
 """
-function discr_bloat_firstorder(cont_sys::AbstractContinuousSystem,
+function discr_bloat_firstorder(cont_sys::InitialValueProblem{<:AbstractContinuousSystem},
                                 δ::Float64)
 
     if !(cont_sys.U isa ConstantNonDeterministicInput)
         error("This discretization algorithm is only implemented for constant inputs")
     end
 
     Anorm = norm(full(cont_sys.s.A), Inf)
-    RX0 = norm(cons_sys.x0, Inf)
-    inputs = next_set(cont_sys.U)
+    RX0 = norm(X0, Inf)
+    inputs = next_set(inputset(cont_sys))
```

Comment:
`next_set` is called `nextinput` now, right?

Reply:
i didn't matter that we kept the name `next_set` inside Reachability and their behavior are slightly different (`nextinput` gives input and state but `next_set` only the first element).

Reply:
I might be wrong, but I think this function is not defined anymore. (You removed the file `Systems.jl`.)

Reply:
`next_set` is defined in `Utils/systems.jl`. at the moment, this file exports

```julia
export ContinuousSystem,
       DiscreteSystem,
       add_dimension,
       next_set
```

Reply:
in never pushed `systems.jl` :D